cognite/neat/rules/models/data_types.py

```diff
@@ -207,7 +222,7 @@ class PlainLiteral(DataType):
     python = str
     dms = dms.Text
     graphql = "String"
-    xsd = "xsd:string"
+    xsd = "xsd:plainLiteral"
```

Comment:
same here, my bad for not suggesting this earlier when you introduced improved data_type handling!